torch/testing/_comparison.py

```diff
@@ -1297,6 +1299,20 @@ def assert_close(
         Traceback (most recent call last):
         ...
         AssertionError: Argh, the tensors are not close!
+        >>> # If msg is a callable, it can be used to augment the generated message with
+        >>> # extra information
+        >>> torch.testing.assert_close(actual, expected, msg=lambda msg: f"Header\n\n{msg}\n\nFooter")
```

Comment:
This line is too long and will be wrapped in the rendered documentation. I have a local commit that fixes this. I'll push if the PR is otherwise approved.